backend/src/main/java/de/bund/digitalservice/ris/norms/domain/entity/Dokument.java

```diff
@@ -287,6 +288,24 @@ public void deleteTimeBoundary(TimeBoundaryChangeData timeBoundaryToDelete) {
     temporalDataNode.removeChild(temporalGroupNode);
   }
 
+  /**
+   * Load the filenames of all Dokumente references by this one.
+   * @return a List of filenames
+   */
+  public List<String> getReferencedDokumenteNames() {
```

Comment:
This is not something that would only belong to the `Rechtsetzungsdokument`?

Reply:
In a norm with a Eingebundene Stammform the e.g. regelungstext-2 is referenced using a componentRef from the regelungstext-1. So also other Dokumente can reference Dokumente.